config.js

```diff
@@ -51,6 +51,7 @@ Options:
 	--s3_force_path_style  Whether to force path style URLs for S3 objects. (default: false)
 	--s3_secret_key <secret>	S3 secret key, required if --s3_endpoint is set. (default: none) 
 	--s3_signature_version <version>	S3 signature version. (default: 4)
+	--s3_acl <canned-acl> S3 object acl. (deafult: public-read)
```

Comment:
Small typo: _deafult_ instead of **default**.

Reply:
Patched in the main branch: https://github.com/OpenDroneMap/NodeODM/commit/d25627b36d1488bd2eb572bf96350fbfd47e5f30